cumulus/bridges/primitives/header-chain/Cargo.toml

```diff
@@ -33,6 +33,7 @@ hex-literal = "0.4"
 default = [ "std" ]
 std = [
 	"bp-runtime/std",
+	"bp-test-utils/std",
```

Comment:
Is it indended that the dev dependency feature is in the list?

Reply:
I think with `resolver = "2"` it should be fine: https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2  
>Features enabled on `dev-dependencies` will not be unified when those same dependencies are used as a normal dependency, unless those dev-dependencies are currently being built.

Reply:
The docs mean it would be ok to enable the feature directly in `dev-dependencies` and not propagate to `features` section. Which I think is what wanted here unless we need to run tests with both `std` feature enabled and disabled?
But indeed, it doesn't pull the dev-dependency when std feature is enabled for `build`.

Reply:
> Which I think is what wanted

This is already the case since we don't specify `default-features = false` on this `bp-test-utils`. In that case, specifying `"bp-test-utils/std"` is harmless, but useless.

Reply:
Thanks, there were quite a few. Removed in https://github.com/paritytech/polkadot-sdk/pull/1194/commits/eddb4ae6b2f2d693398e0204c9d1153e47b1c8d0.  
CI now takes this into account.